providers/openshift.md

```diff
@@ -32,9 +41,146 @@ master_hostname
 node_hostname
 ```
 
-openshift_use_manageiq is set to True to configure the Service Account needed by ManageIQ. there are more like variables like use_metrics to enable other options in OpenShift.
-A more detailed example will be found in inventory/byo/hosts.example.
+openshift_use_manageiq is set to True to configure the Service Account needed
+by ManageIQ. there are more like variables like use_metrics to enable other
+options in OpenShift.  A more detailed example will be found in
+inventory/byo/hosts.example.
```

Comment:
Not really related to your addition but it would be nice to change a couple of things that changed if you have the time.
`openshift_use_manageiq` is no longer needed as it is set by default in openshift-ansible[1]
I think we can update the above paragraph to:

A more detailed example can be found in inventory/byo/hosts.origin.example. Specifically, to deploy metrics and logging use `openshift_hosted_metrics_deploy=True` and `openshift_hosted_logging_deploy=True`.